### PR TITLE
JW-Entity-Framework

### PR DESCRIPTION
**Pull request by JasonWarrenUK**
## Description

IS DATABASE

INSTALL PSQL

THEN

SEE DATABASE

## Type of change

- [ ] ~~Bug fix~~
- [ ] ~~New feature~~
- [ ] ~~Refactoring~~
- [ ] ~~Documentation update~~
- [ ] Other (please describe): IS DATABASE

## How has this been tested?

1. Crying
2. Swearing
3. Getting a new computer
4. Installing Xcode
5 to 84. PATH PATH PATH PATH
85. IS DATABASE

## Checklist:

- [NO] I have reviewed the code changes.
- [NO] I have tested the code changes.
- [NO] I have updated documentation accordingly.
- [NO] My changes do not break existing functionality.

## Additional comments

IS DATABASE

### CODE REVIEW

File: Data/DbContext.cs [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
 {
-    public class ApplicationDbContext : DbContext
+    public class ColourContext : DbContext
     {
-        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
-            : base(options) { }
+        public ColourContext(DbContextOptions<ColourContext> options)
+            : base(options)
+        {
+        } //todo What's this?
```
Comment (JoshCodedit) [inline Data/DbContext.cs:12]: If it helps I can make another PR and add some comments to this file to make your life easier. Let me know

File: appsettings.json [hunk 2]
```diff
@@ -7,6 +7,6 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "DefaultConnection": "Host=localhost;Port=5432;Database=TripleJDatabase;Username=yourUsername;Password=yourPassword"
+    "DefaultConnection": "Host=localhost;Port=5432;Database=triplejdatabase;Username=jasonWarren;Password=KaliLillith@Postgresql13"
```
Comment (jackcasstlesjones) [inline appsettings.json:10]: can we please get a comment on this?

File: Program.cs [hunk 3]
```diff
@@ -1,11 +1,17 @@
 using Microsoft.EntityFrameworkCore;
 using J3.Data;
 using J3.Routes;
+using J3.Data;
```
Comment (JoshCodedit) [inline Program.cs:4]: J3.Data referenced twice here

**Review by jackcasstlesjones — CHANGES_REQUESTED** [review]
Just one request for a comment and then happy to approve!

File: appsettings.json (hunk 2 above)
Comment (jackcasstlesjones) [reply appsettings.json:10]: Just to explain exactly what it goes

File: Data/DbContext.cs (hunk 1 above)
Comment (JasonWarrenUK) [reply Data/DbContext.cs:12]: All good, it was a note to self

File: Program.cs (hunk 3 above)
Comment (JasonWarrenUK) [reply Program.cs:4]: Thankyou!

**Review by jackcasstlesjones — APPROVED** [review]

**Pull request merged**